WordPress/twentytwentyfour
# Add duotone presets

**Description**
Adds duotone presets based on the existing beautiful color palette.  Also removes the default duotones, to promote a more cohesive design experience with the theme. 

Aside: Is is possible to reference the color presets, instead of the hex values, in duotone JSON?

**Screenshots**

<img width="1225" alt="CleanShot 2023-08-28 at 13 13 00" src="https://github.com/WordPress/twentytwentyfour/assets/1813435/12d9ac46-ae25-4841-8a84-6415acf2945e">

<img width="278" alt="CleanShot 2023-08-28 at 13 15 39" src="https://github.com/WordPress/twentytwentyfour/assets/1813435/f2b34c8b-8a14-4607-b816-e656c5dc097a">

**Testing Instructions**


1. Activate the theme.
2. Add an image.
3. See duotone presets.
